product_docs/docs/pgd/5/node_management/heterogeneous_clusters.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Joining a heterogeneous cluster
+---
+
+### Joining a heterogeneous cluster
+
+PGD 4.0 node can join a EDB Postgres Distributed cluster running 3.7.x at a specific
```

Comment:
Awkward grammar. Also... I think we talked about this, but can we add a small note at the top here on the relevance of this topic to PGD 5 (upgrades)?

Reply:
Still waiting for a nod to take this out and move it to upgrading. It's untouched content.

Reply:
Yes, please move it into the upgrading section  (especially since it's version-specific)